### DISCUSSION TITLE
Initialization failed. Flow task start aborted. Loading reduced web interface...

### DISCUSSION THREAD

**Discussion by nandreas777** [jomjol/AI-on-the-edge-device#2514] [Q&A] [answered]
Hallo, habe 2 ESP32-CAM-MB besorgt und über Web-Installer installiert.
SD-Karte von Hand Kopiert. Kommt folgende Meldung :

I (27) boot: ESP-IDF 4.3.2 2nd stage bootloader
I (27) boot: compile time 20:02:47
I (27) boot: chip revision: 3
I (30) boot_comm: chip revision: 3, min. bootloader chip revision: 0
I (37) boot.esp32: SPI Speed      : 40MHz
I (42) boot.esp32: SPI Mode       : DIO
I (46) boot.esp32: SPI Flash Size : 4MB
I (51) boot: Enabling RNG early entropy source...
I (56) boot: Partition Table:
I (60) boot: ## Label            Usage          Type ST Offset   Length
I (67) boot:  0 nvs              WiFi data        01 02 00009000 00004000
I (75) boot:  1 otadata          OTA data         01 00 0000d000 00002000
I (82) boot:  2 phy_init         RF data          01 01 0000f000 00001000
I (89) boot:  3 ota_0            OTA app          00 10 00010000 001db000
I (97) boot:  4 ota_1            OTA app          00 11 001f0000 001db000
I (104) boot: End of partition table
I (109) boot_comm: chip revision: 3, min. application chip revision: 0
I (116) esp_image: segment 0: paddr=001f0020 vaddr=3f400020 size=4634ch (287564) map
I (235) esp_image: segment 1: paddr=00236374 vaddr=3ffb0000 size=03ce0h ( 15584) load
I (241) esp_image: segment 2: paddr=0023a05c vaddr=40080000 size=05fbch ( 24508) load
I (252) esp_image: segment 3: paddr=00240020 vaddr=400d0020 size=1558e0h (1399008) map
I (787) esp_image: segment 4: paddr=00395908 vaddr=40085fbc size=14494h ( 83092) load
I (837) boot: Loaded app from partition at offset 0x1f0000
I (837) boot: Disabling RNG early entropy source...
I (849) quad_psram: This chip is ESP32-D0WD
I (849) esp_psram: Found 8MB PSRAM device
I (849) esp_psram: Speed: 40MHz
I (849) esp_psram: PSRAM initialized, cache is in low/high (2-core) mode.
W (856) esp_psram: Virtual address not enough for PSRAM, map as much as we can. 4MB is mapped
I (865) cpu_start: Pro cpu up.
I (867) cpu_start: Starting app cpu, entry point is 0x40082dbc
I (0) cpu_start: App cpu up.
I (1761) esp_psram: SPI SRAM memory test OK
I (1770) cpu_start: Pro cpu start user code
I (1770) cpu_start: cpu freq: 160000000 Hz
I (1771) cpu_start: Application information:
I (1771) cpu_start: Project name:     AI-on-the-edge
I (1776) cpu_start: App version:      v15.2.4
I (1780) cpu_start: Compile time:     May  2 2023 05:56:12
I (1785) cpu_start: ELF file SHA256:  3bf7289bb70a59d5...
I (1790) cpu_start: ESP-IDF:          5.0.1
I (1794) cpu_start: Min chip rev:     v0.0
I (1798) cpu_start: Max chip rev:     v3.99 
I (1802) cpu_start: Chip rev:         v3.1
I (1806) heap_init: Initializing. RAM available for dynamic allocation:
I (1812) heap_init: At 3FFAE6E0 len 00001920 (6 KiB): DRAM
I (1817) heap_init: At 3FFBD888 len 00022778 (137 KiB): DRAM
I (1822) heap_init: At 3FFE0440 len 00003AE0 (14 KiB): D/IRAM
I (1828) heap_init: At 3FFE4350 len 0001BCB0 (111 KiB): D/IRAM
I (1834) heap_init: At 4009A450 len 00005BB0 (22 KiB): IRAM
I (1839) esp_psram: Adding pool of 4084K of PSRAM memory to heap allocator
I (1847) spi_flash: detected chip: generic
I (1849) spi_flash: flash io: dio
I (1864) cpu_start: Starting scheduler on PRO CPU.
I (0) cpu_start: Starting scheduler on APP CPU.
I (1865) esp_psram: Reserving pool of 40K of internal memory for DMA/internal allocations
I (1865) MAIN: 

================ Start app_main =================
I (1985) MAIN: =================================================
I (2005) MAIN: ==================== Start ======================
I (2025) MAIN: =================================================
I (2045) MAIN: PSRAM size: 8388608 byte (8MB / 64MBit)
I (2065) MAIN: Total heap: 4377083 byte
I (2085) gpio: GPIO[32]| InputEn: 0| OutputEn: 1| OpenDrain: 0| Pullup: 0| Pulldown: 0| Intr:0 
I (4085) gpio: GPIO[25]| InputEn: 1| OutputEn: 0| OpenDrain: 0| Pullup: 1| Pulldown: 0| Intr:2 
I (4085) cam_hal: cam init ok
I (4085) sccb: pin_sda 26 pin_scl 27
I (4085) sccb: sccb_i2c_port=1

I (4085) gpio: GPIO[32]| InputEn: 0| OutputEn: 1| OpenDrain: 0| Pullup: 0| Pulldown: 0| Intr:0 
I (4125) camera: Detected camera at address=0x30
I (4125) camera: Detected OV2640 camera
I (4125) camera: Camera PID=0x26 VER=0x42 MIDL=0x7f MIDH=0xa2
I (4215) cam_hal: buffer_size: 32768, half_buffer_size: 4096, node_buffer_size: 2048, node_cnt: 16, total_cnt: 15
I (4215) cam_hal: Allocating 61440 Byte frame buffer in PSRAM
I (4215) cam_hal: cam config ok
I (4215) ov2640: Set PLL: clk_2x: 0, clk_div: 0, pclk_auto: 0, pclk_div: 8
I (6295) MAIN: Camera info: PID: 0x26, VER: 0x42, MIDL: 0x7f, MIDH: 0xa2
I (6315) SDCARD: Basic R/W check started...
I (6385) SDCARD: Basic R/W check successful
I (6465) SNTP: TimeServer not defined, using default: pool.ntp.org
I (6485) SNTP: Configuring NTP Client...
I (6525) SNTP: Time zone set to CET-1CEST,M3.5.0,M10.5.0/3
I (6545) SNTP: The local time is unknown, starting with 1970-01-01 01:00:04
I (6555) SNTP: Once the NTP server provides a time, we will switch to that one
I (6595) MAIN: CPU frequency: 160 MHz
I (6625) OTA: Start CheckOTAUpdateCheck...
I (6635) OTA: SHA-256 for the partition table: : 9d4a2809ae90fb20709a8ca3dd3ac3462e1e73286bcd31b689060c91de06fa3a
I (6655) OTA: SHA-256 for bootloader: : 4a9573dafac5d5c79c43e5a356568332da3a99dd2e220da6ceeadc96d32708a2
I (7135) OTA: SHA-256 for current firmware: : b466e493336ff9d3c9569adde1350c625cd15e53b5a82ea56fa857d3ba3d3945
I (7175) SDCARD: Folder/file presence check started...
E (7275) SDCARD: Folder/file check: File /html/version.txt not found
E (7295) HELPER: New System Status: 0x00000010
I (7335) MAIN: Tag: 'v15.2.4', Release: v15.2.4 (Commit: cbd14a2), Date/Time: 2023-05-02 05:56, Web UI: ?
W (7385) MAIN: Failed to read file html/version.txt to parse Web UI version
W (7445) MAIN: Web UI version (?) does not match firmware version (cbd14a2)
W (7465) MAIN: Recommendation: Repeat installation using AI-on-the-edge-device__update__*.zip
I (7495) MAIN: Reset reason: Power-on event (or reset button)
I (7515) WLANINI: SSID: FRITZ!Box 7580 LO
I (7535) WLANINI: Password: XXXXXXXX
I (7555) WLANINI: Hostname: gasmeter
I (7575) WLANINI: RSSIThreshold: 0
I (7605) MAIN: WLAN config loaded, init WIFI...
I (7625) WIFI: Automatic interface config --> Use DHCP service
I (7655) wifi:wifi driver task: 3ffd1704, prio:23, stack:6144, core=0
I (7655) system_api: Base MAC address is not set
I (7655) system_api: read default base MAC address from EFUSE
I (7675) wifi:wifi firmware version: 17afb16
I (7675) wifi:wifi certification version: v7.0
I (7675) wifi:config NVS flash: enabled
I (7675) wifi:config nano formating: enabled
I (7675) wifi:Init data frame dynamic rx buffer num: 16
I (7685) wifi:Init management frame dynamic rx buffer num: 16
I (7685) wifi:Init management short buffer num: 32
I (7695) wifi:Init static tx buffer num: 16
I (7695) wifi:Init tx cache buffer num: 16
I (7705) wifi:Init static rx buffer size: 1600
I (7705) wifi:Init static rx buffer num: 16
I (7705) wifi:Init dynamic rx buffer num: 16
I (7715) wifi_init: rx ba win: 16
I (7715) wifi_init: tcpip mbox: 32
I (7715) wifi_init: udp mbox: 6
I (7725) wifi_init: tcp mbox: 6
I (7725) wifi_init: tcp tx win: 5744
I (7725) wifi_init: tcp rx win: 5744
I (7735) wifi_init: tcp mss: 1440
I (7735) wifi_init: WiFi/LWIP prefer SPIRAM
I (7735) wifi_init: WiFi IRAM OP enabled
I (7745) wifi_init: WiFi RX IRAM OP enabled
I (7765) phy_init: phy_version 4670,719f9f6,Feb 18 2021,17:07:07
I (7865) wifi:mode : sta (a0:b7:65:50:7f:38)
I (7865) wifi:enable tsf
I (7875) WIFI: Set hostname to: gasmeter
I (7895) WIFI: Init successful
I (10285) wifi:new:<1,0>, old:<1,0>, ap:<255,255>, sta:<1,0>, prof:11
I (11005) wifi:state: init -> auth (b0)
I (11015) wifi:state: auth -> init (8a0)
I (11015) wifi:new:<1,0>, old:<1,0>, ap:<255,255>, sta:<1,0>, prof:11
I (11015) wifi:new:<1,0>, old:<1,0>, ap:<255,255>, sta:<1,0>, prof:11
I (11015) wifi:state: init -> auth (b0)
I (11035) wifi:state: auth -> assoc (0)
I (11045) wifi:state: assoc -> run (10)
I (11095) wifi:connected with FRITZ!Box 7580 LO, aid = 10, channel 1, BW20, bssid = 48:5d:35:40:6b:ff
I (11095) wifi:security: WPA2-PSK, phy: bgn, rssi: -68
I (11105) wifi:pm start, type: 1

I (11105) WIFI: Connected to: FRITZ!Box 7580 LO, RSSI: -68
I (11125) wifi:<ba-add>idx:0 (ifx:0, 48:5d:35:40:6b:ff), tid:6, ssn:2, winSize:64
I (11145) wifi:AP's beacon interval = 102400 us, DTIM period = 1
I (12105) esp_netif_handlers: sta ip: 192.168.178.181, mask: 255.255.255.0, gw: 192.168.178.1
I (12105) WIFI: Assigned IP: 192.168.178.181
I (27915) MAIN: Device info: CPU cores: 2, Chip revision: 301
I (27935) MAIN: SD card info: Name: SA16G, Capacity: 14760MB, Free: 14730MB
I (29955) MAIN SERVER: Starting server on port: '80'
I (29955) MAIN SERVER: Registering URI handlers
I (29955) MAINCTRL: server_main_flow_task - Registering URI handlers
I (29955) OTA: Registering URI handlers
I (29955) GPIO: start GpioHandler
I (29965) GPIO: register GPIO Uri
I (29965) GPIO: server_GPIO - Registering URI handlers
E (29965) MAIN: Initialization failed. Flow task start aborted. Loading reduced web interface...
E (36075) MAIN SERVER: We have a critical error, not serving main page!

Bei beiden Boards das selbe Problem, kann mir bitte jemand da weiter helfen ?

**Comment by friedpa** [comment]
Bist Du dir sicher, dass die Firmware SW zu der Version der Web UI Files auf der SD Card passt?

W (7385) MAIN: Failed to read file html/version.txt to parse Web UI version
W (7445) MAIN: Web UI version (?) does not match firmware version (https://github.com/jomjol/AI-on-the-edge-device/commit/cbd14a267f1b31b6feaed63fe7725f1d89320139)
W (7465) MAIN: Recommendation: Repeat installation using AI-on-the-edge-device__update__*.zip

**Reply by nandreas777** [reply]
Wie sehe ich denn das , oder wie überprüfe ich es , bin leider noch ein Neuling.

**Comment by friedpa** [comment]
Mache nochmals einen OTA Update. Die neueste Rolling findest Du in Github unter Actions. Klicke auf den Titel des letzten Rolling, dann sieht Du untenstehend die verfügbaren Versionen. Nimm die "update" version und spiele diese über die Web UI ein. Sollte das nicht funktionieren, dann musst Du das händisch machen (die HTML Files auf die SD Karte kopieren).

**Reply by nandreas777** [reply, marked as answer]
Habe es mit update version gemacht und es funktioniert , Danke !!!

**Reply by caco3** [reply]
Hinweis: Anstatt `rolling` sollten Anfänger besser das letzte Release nehmen!
